src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { Comment } from '../../types/Comment';
+import { getPostComments } from '../../api/comments';
+
+type InitialState = {
+  loaded: boolean;
+  hasError: boolean;
+  items: Comment[];
+};
+
+const initialState: InitialState = {
+  loaded: false,
```

Comment:
**Loaded** means that the process of loading is already finished so I guess here you wanted to call the property in state **`loading`**

And pls do the same for all such cases in the project, where `loading` should be used instead of `loaded`